### PR TITLE
aarohmankad's first goodie

### PR DESCRIPTION
**Pull request by aarohmankad**
**What does your Instant Answer do?** It lets people know I'm awesome and that I'm ready to contribute to DuckDuckGo in the future

**What problem does your Instant Answer solve (Why is it better than organic links)?** It tells anyone who searches this that I'm awesome.

**What is the data source for your Instant Answer? (Provide a link if possible)** No Data Source needed.

**Why did you choose this data source?** N/A

**Are there any other alternative (better) data sources?** N/A

**What are some example queries that trigger this Instant Answer?** duckduckhack aarohmankad

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)** My huge fanbase jk.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?** No

**Which existing Instant Answers will this one supercede/overlap with?** None like this exist.

**Are you having any problems? Do you need our help with anything?** Nope.

**Where did you hear about DuckDuckHack? (For first time contributors)** I really liked the search engine, I saw a contribute button somewhere.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)** screenshot 10

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [x] Google Chrome
        - [x] Firefox
        - [x] Opera
        - [x] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [x] Firefox
        - [x] Native Browser
        - [x] Google Chrome
        - [x] Opera

```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/aarohmankad.pm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+attribution github => ["http://www.github.com/aarohmankad", "Aaroh Mankad"],
+            twitter => "aarohmankad";
+
+
+triggers start => "duckduckhack aarohmankad";
+
+
+handle remainder => sub {
+    return if $_;
+	return "aarohmankad is awesome and has successfully completed the DuckDuckHack Goodie tutorial!";
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/aarohmankad.pm]: Can you fix this hard-tab? It should be 4 spaces :+1:

**Comment by mintsoft** [comment]
Congratulations on your first goodie @aarohmankad. There's just one indentation problem, once that's fixed we'll get it merged 'n' live! :tada:

**Comment by aarohmankad** [comment]
That should fix it.

**Comment by moollaza** [comment]
@aarohmankad congrats! It's live: https://duckduckgo.com/?q=duckduckhack+aarohmankad&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**